docs/introduction/memory-cache.md

```diff
@@ -0,0 +1,60 @@
+# Memory Cache
```

Comment:
this file should be mentioned in `docs/introduction/navigation.yml` and `docs/introduction/index.md`